modules/apps/27-interchain-accounts/host/keeper/handshake_test.go

```diff
@@ -47,16 +47,19 @@ func (suite *KeeperTestSuite) TestOnChanOpenTry() {
 		path     *ibctesting.Path
 		chanCap  *capabilitytypes.Capability
 		metadata icatypes.Metadata
+		version  string
 	)
 
 	testCases := []struct {
 		name     string
 		malleate func()
+		assert   func()
```

Comment:
You shouldn't add an assert function since you can just check the `host_connection_id` after every successful handshake anyway.